example-application/api-under-test.js

```diff
@@ -79,6 +79,60 @@ const defineRoutes = (expressApp) => {
     });
   });
 
+  /**
+   * Pubsub API
+   */
+
+  const topicName = 'test-topic';
+  const subscriptionName = 'test-subscription';
+  const pubsubHandler = new PubsubHandler();
+  const instanceMessages = [];
+
+  // send a message to the pubsub
+  pubsubRouter.post('/emit', async (req, res, next) => {
```

Comment:
As a principle - Do we want to recommend for people who work with message queues to include routes for them? what if they don't have API at all? If we want to emit messages, why wouldn't the test approach the MQ wrapper?

Reply:
Removing this soon!